### PR TITLE
Fixed vouching for Plast Member and added some new roles to vouch. Fixed size for details on Governing Body. Fixed "Confirmed" field

### PR DESCRIPTION
**Pull request by nicoolle**
dev
## JIRA

* [Main JIRA ticket](https://jira.softserve.academy/secure/RapidBoard.jspa?rapidView=id)

## Code reviewers

- [ ] @github_username

### Second Level Review

- [ ] @github_username

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

File: src/api/decisionsApi.tsx [hunk 1]
```diff
@@ -55,86 +53,85 @@ export type DecisionPost  ={
...
-    for (let i = 0; i < binaryLen; i += 1) {
-      const ascii = binaryString.charCodeAt(i);
-      bytes[i] = ascii;
-      };
-    const blob = new Blob([bytes], {type: "application/pdf"});
-    const link = window.URL.createObjectURL(blob);
-    return  link;
+const getAll = async () => {
+  const { data } = await Api.get("Decisions");
+
```
Comment (OleksandrBilan) [inline src/api/decisionsApi.tsx:88]: Why do you need this blank line?

File: src/api/decisionsApi.tsx [hunk 2]
```diff
@@ -55,86 +53,85 @@ export type DecisionPost  ={
...
+      searchedData: searchedData,
+      page: page,
+      pageSize: pageSize,
+    })).data;
+};
+
+const getOnCreate = async () => {
+  const data: DecisionOnCreateData = await (await Api.get(`Decisions/NewDecision`)).data;
+  return data;
+};
```
Comment (OleksandrBilan) [inline src/api/decisionsApi.tsx:104]: add blank line between different functions, please

**Review by OleksandrBilan — COMMENTED** [review]
Please, separate each function declaration with blank line

File: src/api/decisionsApi.tsx (hunk 2 above)
Comment (nicoolle) [reply src/api/decisionsApi.tsx]: done

File: src/api/decisionsApi.tsx (hunk 1 above)
Comment (nicoolle) [reply src/api/decisionsApi.tsx]: done

File: src/api/decisionsApi.tsx [hunk 3]
```diff
@@ -62,7 +60,7 @@ export type DecisionPost  ={
   };
   export const statusTypeGetParser = (statusType: number): string =>{
     if(statusType === 0) return "У розгляді";
-    if (statusType === 1) return "Підтверджено";
+    if (statusType === 1) return "Потверджено";
```
Comment (ostappppppp) [inline src/api/decisionsApi.tsx:63]: just out of curious, is this really how Plast call something is approved?😂😂😂

Comment (nicoolle) [reply src/api/decisionsApi.tsx:63]: Yes, this is Old Slavic or something like that))

**Review by christykobyletska — APPROVED** [review]

File: src/api/decisionsApi.tsx (hunk 3 above)
Comment (ostappppppp) [reply src/api/decisionsApi.tsx:63]: Got you
For we it's really weird

Comment (nicoolle) [reply src/api/decisionsApi.tsx:63]: Maybe you want also review and approve this pull request?)

Comment (ostappppppp) [reply src/api/decisionsApi.tsx:63]: Sure

**Review by ostappppppp — APPROVED** [review]

**Review by OleksandrBilan — APPROVED** [review]

**Pull request merged**